test/e2e/network/service.go

```diff
@@ -1623,6 +1623,126 @@ var _ = SIGDescribe("ESIPP [Slow] [DisabledForLargeClusters]", func() {
 		}
 	})
 
+	It("should be able to enable session affinity for NodePort service", func() {
```

Comment:
Oh, didn't realize you put all these It blocks under "ESIPP [Slow]". We should move them elsewhere (to above "services" block?), otherwise they will not run in presubmit.

Reply:
Moved.